hw3/asdetmin/.gitignore

```diff
@@ -0,0 +1,9 @@
+/.bundle/
+/.yardoc
+/Gemfile.lock
```

Comment:
mm, seems like it was default option that was applyed after gem generation command running. I actually never get ignore Gemfile.lock to avoid the problems with versioning and deployment